WORKSPACE

```diff
@@ -68,9 +61,12 @@ http_file(
 )
 
 # For Jetty
-http_file(
+new_http_archive(
    name = "jetty",
+   build_file = "BUILD.jetty",
    url = "http://central.maven.org/maven2/org/eclipse/jetty/jetty-distribution/9.4.4.v20170414/jetty-distribution-9.4.4.v20170414.tar.gz",
+   strip_prefix = "jetty-distribution-9.4.4.v20170414/",
```

Comment:
Consider hoisting a single version constant?

I did this myself here: https://github.com/bazelbuild/rules_docker/blob/master/docker/docker.bzl#L28